pyproject.toml

```
@@ -0,0 +1,64 @@
+[project]
+name = "lighthouse"
+version = "0.1.0a1"
+requires-python = ">=3.10,<3.13"  # Bounds are due to torch-mlir's packaging
+dependencies = [
+    "mlir-python-bindings==20251011+3af43e303",
+]
+
+[project.optional-dependencies]
+# Additional "targets" which pull in optional dependencies -- use `uv sync --extra TARGET`
+ingress_torch_cpu = [
+    "torch==v2.8.0+cpu",
+    "torch-mlir==20251003.589"
+]
+ingress_torch_nvidia = [
+    "torch==2.8.0",  # Nvidia-enabled version of torch
```

[nit] If possible, for consistency sake (*), could you keep put comments on separate lines?
```suggestion
     # Nvidia-enabled version of torch
    "torch==2.8.0",
```

(*) This file mixes two styles ATM.